### ISSUE TITLE
Module doesn't compile with DRM changes in Linux 4.5

### ISSUE THREAD

**Issue by dansiemon** [DisplayLink/evdi#8]
There are kernel DRM API changes in the upcoming 4.5 kernel. The evdi module doesn't compile with these changes.

**Comment by ajbogh** [comment]
That issue is now fixed from this PR #10 . It has been merged today.

The changes don't seem to fix the display issue, but USB3 seems to work, network works as well. The network may become available after some time, however it seems to start sooner if you unplug the DisplayLink and plug it back into the USB port.

```
[131958.102309] IPv6: ADDRCONF(NETDEV_UP): enx6c0b842122e1: link is not ready
[131961.093188] cdc_ncm 2-2.1:1.5 enx6c0b842122e1: 1000 mbit/s downlink 1000 mbit/s uplink
[131961.109276] cdc_ncm 2-2.1:1.5 enx6c0b842122e1: network connection: connected
[131961.109338] IPv6: ADDRCONF(NETDEV_CHANGE): enx6c0b842122e1: link becomes ready
```

**Comment by displaylink-mlukaszek** [comment]
Thanks! We'll track the video function separately - this issue was only about compilation failing.
